senior-knights/course-schedulizer
# Add About Page

This PR adds the About Page according to the [specifications on the CS 396 website](https://cs.calvin.edu/courses/cs/396/website.html). 

*To Test*
1. `npm i` and `npm start`
1. I recommend opening up the link to the CS website and look at that while you read through the About Page. Please check the grammar and please let me know if you want anything changed! You could even make a "suggested changes" thing if you want. After that looks good, then you can look at the code to make sure that looks all good. Thanks for the reviews.

*Components added*
- the about page
- text sections for the about page
- animated titles
- animated imaged you can mouse over
- a footer

This was a bigger PR than I was hoping for. So I apologize for that. 

When this is merged, then we can add a link to the About Page to the class Moodle.

closes #72
